canonical-web-and-design/charmhub.io
# Holding page

## Done

- A quick holding page based on https://zpl.io/2GJmBzr
- Update linting on commit to use the dotrun snap
- copy the juju.is suru strip

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8045 see the holding page
